spec/test-utils/test-data/static-test-data.ts

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Missing copyright

Reply:
Removed this file to use the generate-test-data.py script